tools/scripts/acr-scan.ps1

```diff
@@ -0,0 +1,97 @@
+<#
+ .SYNOPSIS
+    Returns relevant scan results for a ACR registry.
+
+ .DESCRIPTION
+    Returns relevant scan results for a ACR registry
+    which can be further processed by another script
+    or converted to json by piping it to ConvertTo-Json
+    commandlet. The script requires az (AzureCLI) to 
+    be installed and you must be logged in (az login)
+
+ .PARAMETER Registry
+    The name of the registry
+
+ .PARAMETER Subscription
+    The subscription to use - otherwise uses default.
+
+ .PARAMETER All
+    Include also vulnerabilities that are not patchable.
+#>
+
+Param(
+    [string] $Registry = "industrialiotprod",
+    [string] $Subscription = "IOT_GERMANY",
+    [switch] $All
+)
+
+# set default subscription
+if (![string]::IsNullOrEmpty($script:Subscription)) {
+    Write-Debug "Setting subscription to $($script:Subscription)"
+    $argumentList = @("account", "set", "--subscription", $script:Subscription, "-ojson")
+    & "az" $argumentList 2`>`&1 | ForEach-Object { "$_" }
+    if ($LastExitCode -ne 0) {
+        throw "az $($argumentList) failed with $($LastExitCode)."
+    }
+}
+
+# get registry resource info
+$argumentList = @("acr", "show", "--name", $script:Registry, "-ojson")
+$result = (& "az" $argumentList)
+if ($LastExitCode -ne 0) {
+    throw "az $($argumentList) failed with $($LastExitCode)."
+}
+$acr = $result | ConvertFrom-Json
+
+# Get all vulnerability assessments
+$argumentList = @("security", "sub-assessment", "list", "-ojson",
+    "--assessed-resource-id", $acr.id,
```

Comment:
"-o json"? json is default anyhow.

Reply:
You can set another default as azure cli configuration, then json is not anymore.